lib/rucio/core/replica.py

```diff
@@ -3607,16 +4025,19 @@ def get_RSEcoverage_of_dataset(scope, name, *, session: "Session"):
     :return:                  Dictionary { rse_id : <total bytes present at rse_id> }
     """
 
-    query = session.query(models.RSEFileAssociation.rse_id, func.sum(models.DataIdentifierAssociation.bytes))
-
-    query = query.filter(and_(models.DataIdentifierAssociation.child_scope == models.RSEFileAssociation.scope,
-                              models.DataIdentifierAssociation.child_name == models.RSEFileAssociation.name,
-                              models.DataIdentifierAssociation.scope == scope,
-                              models.DataIdentifierAssociation.name == name,
-                              models.RSEFileAssociation.state != ReplicaState.BEING_DELETED,
-                              ))
-
-    query = query.group_by(models.RSEFileAssociation.rse_id)
+    stmt = select(
+        models.RSEFileAssociation.rse_id,
+        func.sum(models.RSEFileAssociation.bytes)
```

Comment:
this is different from the previous query, where `models.DataIdentifierAssociation.bytes` was used instead of `models.RSEFileAssociation.bytes`